apps/confidential/confidential.go

```diff
@@ -163,12 +167,12 @@ func NewCredFromAssertion(assertion string) (Credential, error) {
 	if assertion == "" {
 		return Credential{}, errors.New("assertion can't be empty string")
 	}
-	return NewCredFromAssertionCallback(func(context.Context) (string, error) { return assertion, nil }), nil
+	return NewCredFromAssertionCallback(func(context.Context, AssertionRequestOptions) (string, error) { return assertion, nil }), nil
 }
 
 // NewCredFromAssertionCallback creates a Credential that invokes a callback to get assertions
 // authenticating the application. The callback must be thread safe.
-func NewCredFromAssertionCallback(callback func(context.Context) (string, error)) Credential {
+func NewCredFromAssertionCallback(callback func(context.Context, AssertionRequestOptions) (string, error)) Credential {
```

Comment:
I am not familiar with Go, but please ensure this does not break backwards compat.

Reply:
This comment apparently landed on the diff between two of this PR's commits. There's no compatibility risk here because `NewCredFromAssertionCallback` is entirely new.